.spin/cmds.py

```diff
@@ -0,0 +1,28 @@
+"""Additional Command-line interface for spin."""
+import os
+import sys
+
+import click
+from spin import util
+from spin.cmds import meson
+
+
+@click.command()
+@click.argument("asv_args", nargs=-1)
+def asv(asv_args):
```

Comment:
You may also want to look at what we've written for NumPy:

https://github.com/numpy/numpy/blob/main/.spin/cmds.py#L331

Yours is much more straightforward, which I like, but perhaps there are useful features there.

Reply:
that's cool! thanks @stefanv. 

for sure, I will have a closer look later on a new PR. For now, I keep it simple and focus on meson + failing CI's. 

this PR is close to be done. I will tackle / update the documentation on a new PR.

@jarrodmillman and @stefanv, thank you for `spin` tools, really useful, looking forward to add new commands on a new PR. 

ps: Do you know why on Windows `set_num_threads` does not work. it seems that OpenMP use all the threads each times.  Did you encounter this issue?

Reply:
Which `set_num_threads` are you referring to?

Are you looking for something like https://pypi.org/project/threadpoolctl/ perhaps?

Reply:
Sorry, I was referring to our own codebase:  https://github.com/dipy/dipy/blob/79c3939e15529edaae6f1da4c8cb92e7abaf4c10/dipy/utils/omp.pyx#L82

but more generally, do you have issue with openmp with scikit-image or other package ?

Reply:
We haven't had any reports, but I'm not sure anyone is watching that closely on Windows. I think the most likely place where these issues would have been hashed out is in `threadpoolctl`.